### PR TITLE
Create eip155-515.json

### PR DESCRIPTION
**Pull request by neatiomod**

### CODE REVIEW

File: _data/chains/eip155-1071.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://neatio.net",
+  "shortName": "NEAT",
+  "chainId": 515,
+  "networkId": 515,
+  "icon": "Φ",
+  "explorers": [{
+    "name": "Neatio Explorer",
+    "url": "https://scan.neatio.net/",
```
Comment (ligi) [inline _data/chains/eip155-1071.json:21]:
```suggestion
    "url": "https://scan.neatio.net",
```

**Review by ligi — CHANGES_REQUESTED** [review]
ChainID in FileName does not match ChainId in JSON

**Review by p1r5 — COMMENTED** [review]
_data/chains/eip155-515.json

**Review by p1r5 — COMMENTED** [review]
_data/chains/eip155-515.json

**Review by ligi — CHANGES_REQUESTED** [review]
the "icon": "Φ" does not exist

**Comment by arsshh** [comment]
https://techy-india.com/en/payment?registration=true

**Pull request closed without merging**